### PR TITLE
[sese] update usage

### PR DESCRIPTION
**Pull request by SHIINASAMA**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

---

Updated the post-version because I noticed that I forgot to update the usage file.

It was my carelessness, and I'm sorry.

### CODE REVIEW

**Comment by dg0yt** [comment]
> It was my carelessness, and I'm sorry.

No, it was merged carelessly. Please consider the review comments in https://github.com/microsoft/vcpkg/pull/41402.

**Comment by SHIINASAMA** [comment]
@dg0yt Thanks for your reviews. Maybe the previous test was not accurate, it really did not have a pdb file at the time, but it has been resolved now and this patch is not necessary.

File: ports/sese/usage [hunk 1]
```diff
@@ -1,7 +1,4 @@
 sese provides CMake targets:
 
     find_package(sese CONFIG REQUIRED)
-    # build full features
-    target_link_libraries(main PRIVATE Sese::Core)
-    # build only plugin
-    target_link_libraries(main PRIVATE Sese::Plugin)
+    target_link_libraries(main PRIVATE Sese::Core)
```
Comment (dg0yt) [inline ports/sese/usage:4]: Please add a line break at the end of the last line. (In VS Code, it is displayed as a new empty line.)

Also, change the indent to 2 spaces to match tool heuristics output.

Comment (LilyWangLL) [reply ports/sese/usage]: @SHIINASAMA could you please review and apply this suggestion?

Comment (SHIINASAMA) [reply ports/sese/usage]: @LilyWangLL I already did. I just pushed it a little slow ...

**Comment by LilyWangLL** [comment]
Usage passed on x64-windows.

**Review by LilyWangLL — APPROVED** [review]

**Comment by JavierMatosD** [comment]
> It was my carelessness, and I'm sorry.

It was my mistake! Thank you for the contribution!

**Pull request merged**